examples/getting-started/index.ipynb

```diff
@@ -0,0 +1,45 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install notebook dependencies\n",
+    "\n",
+    "```\n",
+    "conda install -c pyviz pyviz\n",
+    "conda install -c intake intake intake-xarray\n",
+    "pip install rasterio\n",
```

Comment:
Why is this installed with pip?

Reply:
Isn't rasterio a dependency we have anyway?

Reply:
I don't think it's part of pyviz currently, but I believe it's part of datashader's doc building, and I don't recall having to install it in some special way other than the usual conda channels for that. I *do* recall there being much pain in getting rasterio working nicely, years ago, but it hasn't been on my radar for a while.

Reply:
Ah, that would be less better.  Also, my conda install of rasterio works now.  I have no idea what the issue was when I tried it before.  I'll update the docs accordingly.

Reply:
Thanks!